megatron/training/arguments.py

```diff
@@ -853,6 +855,8 @@ def _add_network_size_args(parser):
                        action='store_false',
                        help='Disable position embedding. Deprecated: use --position-embedding-type',
                        dest='add_position_embedding')
+    group.add_argument('--use-te-fused-rope', action='store_true', default = True,
```

Comment:
Since the action is `store_true` and the default value is `True`, it seems that there is no way to specify if they don't want to use fused RoPE from TE... Please fix this.